docs/OMF.rst

```diff
@@ -164,7 +172,11 @@ The second screen will request the following information:
    - **Number Format:** Used to match Fledge data types to the data type configured in PI. The defaults is float64 but may be set to any OMF datatype that supports floating point values.
    - **Compression:** Compress the readings data before sending it to the PI System.
 
+<<<<<<< HEAD
```

Comment:
Conflicts needs to be resolved in this file.

Reply:
Thanks had not noticed that.